packages/webpack/css-extract-webpack-plugin/src/CssExtractRspackPlugin.ts

```diff
@@ -10,10 +10,8 @@ import type {
   CssExtractRspackPluginOptions as ExternalCssExtractRspackPluginOptions,
 } from '@rspack/core';
 
-import {
-  LynxEncodePlugin,
-  LynxTemplatePlugin,
-} from '@lynx-js/template-webpack-plugin';
+import { removeFunctionWhiteSpace } from '@lynx-js/css-serializer/dist/plugins/removeFunctionWhiteSpace.js';
```

Comment:
Use

```js
import { CSS } from '@lynx-js/template-webpack-plugin'
CSS.Plugins.removeFunctionWhiteSpace()
```

instead.

`@lynx-js/css-serializer` is a devDependency of `@lynx-js/css-extract-webpack-plugin`.